src/main/src/main/java/org/geoserver/catalog/FixedValueRange.java

```diff
@@ -0,0 +1,111 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.catalog;
+
+import java.text.ParseException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.TreeSet;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.geoserver.ows.kvp.TimeParser;
+import org.geotools.util.logging.Logging;
+
+public class FixedValueRange {
```

Comment:
Please refact a bit this class. If you want to have a Class with parsing capabilities for the fixed value string it's ok, but then avoid such massive usage of static method. For instance you could instead have a constructor that accept the fixed value string and split it. The other methods turned to the class methods could then use the class attribute String [] to work on while doing the parsing job.
Also the name should be changed, I would call it FixedValue parser.